fe/fe-core/src/main/java/com/starrocks/transaction/TransactionState.java

```diff
@@ -398,12 +399,14 @@ public void setTabletCommitInfos(List<TabletCommitInfo> infos) {
 
     public boolean tabletCommitInfosContainsReplica(long tabletId, long backendId) {
         TabletCommitInfo info = new TabletCommitInfo(tabletId, backendId);
-        if (this.tabletCommitInfos == null || this.tabletCommitInfos.contains(info)) {
+        if (this.tabletCommitInfos == null) {
+            Backend backend = GlobalStateMgr.getCurrentSystemInfo().getBackend(backendId);
             // if tabletCommitInfos is null, skip this check and return true
+            LOG.warn("tabletCommitInfos is null in TransactionState, tabletid {} backend {} transid {}",
```

Comment:
```suggestion
            LOG.warn("tabletCommitInfos is null in TransactionState, tablet {} backend {} txn {}",
```

Reply:
we need this fix in next release branch, I will do this in next pr